indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskRunner.java

```diff
@@ -154,5 +146,14 @@ default void updateLocation(Task task, TaskLocation location)
     // do nothing
   }
 
+  default int getTotalCapacity()
```

Comment:
Best add javadocs to these methods esp to point out that they can return -1 when not implemented or can't be determined.